sdk/firstload/src/browserExtension/extensionListener.ts

```diff
@@ -1,4 +1,4 @@
-import { H } from '..'
+import { H } from '../index'
```

Comment:
Similar to https://github.com/highlight/highlight/pull/4586/files#r1167365950, `..` is also not a valid relative specifier.